File: frameworks/Java/fair/java-micronaut-jdbc-repository/src/main/java/io/techempower/benchmark/micronaut/Application.java
package io.techempower.benchmark.micronaut;

import io.micronaut.runtime.Micronaut;

public class Application {

    static void main(String[] args) {
        System.out.println("AVAILABLE CORES: " + Runtime.getRuntime().availableProcessors());
        Micronaut.run(Application.class, args);
    }
}

File: frameworks/Java/fair/kotlin-kora1-vertx-repository-suspend/src/main/kotlin/io/techempower/benchmark/kora/Application.kt
package io.techempower.benchmark.kora

import ru.tinkoff.kora.application.graph.KoraApplication
import ru.tinkoff.kora.common.KoraApp
import ru.tinkoff.kora.config.hocon.HoconConfigModule
import ru.tinkoff.kora.database.vertx.VertxDatabaseModule
import ru.tinkoff.kora.http.server.undertow.UndertowHttpServerModule
import ru.tinkoff.kora.json.module.JsonModule

@KoraApp
interface Application : HoconConfigModule,
    JsonModule,
    VertxDatabaseModule,
    UndertowHttpServerModule

fun main() {
    KoraApplication.run { ApplicationGraph.graph() }
}
